src/main/java/bio/terra/workspace/service/create/CreateDAO.java

```diff
@@ -34,6 +34,14 @@ public String create(String workspaceId, JsonNullable<UUID> spendProfile) {
     }
 
     jdbcTemplate.update(sql, paramMap);
-    return workspaceId;
+    return workspaceId.toString();
+  }
+
+  public boolean deleteWorkspace(UUID workspaceId) {
+    Map<String, Object> paramMap = new HashMap<String, Object>();
+    paramMap.put("id", workspaceId.toString());
+    int rowsAffected =
+        jdbcTemplate.update("DELETE FROM workspace WHERE workspace_id = :id", paramMap);
+    return rowsAffected > 0;
```

Comment:
thinking out loud: we'll have to deal with foreign key constraints and other dependencies once the service supports them. Nothing to do right now.